doc/releases/mimic.rst

```diff
@@ -1,3 +1,162 @@
+v13.2.5 Mimic
+=============
+
+This is the fifth bugfix release of the Mimic v13.2.x long term
+stable release series. We recommend all mimic users upgrade.
+
+Notable Changes
+---------------
+
+* This release fixes the pg log hard limit bug that was introduced in
+  13.2.2, https://tracker.ceph.com/issues/36686. A flag called
+  `pglog_hardlimit` has been introduced, which is off by default. Enabling
+  this flag will limit the length of the pg log.  In order to enable
+  that, the flag must be set by running `ceph osd set pglog_hardlimit`
+  after completely upgrading to 13.2.2. Once the cluster has this flag
+  set, the length of the pg log will be capped by a hard limit. Once set,
+  this flag *must not* be unset anymore. In luminous, this feature was
+  introduced in 12.2.11. Users who are running 12.2.11, and want to
+  continue to use this feauture, should upgrade to 13.2.5 or later.
+
+* This release also fixes a cve on civetweb, CVE-2019-3821 where ssl fds were
+not closed in civetweb in case the initial negotiation fails.
```

Comment:
please indent this line, for consistency with the preceding and subsequent bullet points